google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/apis/ServiceAccountKeyUltimateDisplayDialog.java

```diff
@@ -93,6 +96,17 @@ public ServiceAccountKeyUltimateDisplayDialog(
       scrollPane.setVisible(false);
       runConfigurationTable.setVisible(false);
     }
+
+    runConfigurationTableModel.addTableModelListener(
+        new TableModelListener() {
```

Comment:
lambda

Reply:
also, looks like visibility of the constructor of this class can be package private